Sources/PointFree/Account/Cancel.swift

```diff
@@ -9,63 +9,100 @@ import Prelude
 import Styleguide
 @testable import Tuple
 
-func requireSubscriptionAndOwner<A>(
```

Comment:
General cleanup in this file to use `filterMap` and `filter` instead of more ad-hoc, logic-full transformers.